### PR TITLE
add gh action output for `url` and `url_html`

### PR DESCRIPTION
**Pull request by peterwhitesell**
# Changes
- [x] add github action output for `url` and `url_html` so they can be referenced by other actions in a workflow

### CODE REVIEW

**Comment by dorny** [comment]
@peterwhitesell Thanks for your contribution.
Could you merge the current `main` to your branch and solve the conflicts?
I will merge it afterward.

File: src/main.ts [hunk 1]
```diff
@@ -193,6 +194,8 @@ class TestReporter {
     core.info(`Check run create response: ${resp.status}`)
     core.info(`Check run URL: ${resp.data.url}`)
     core.info(`Check run HTML: ${resp.data.html_url}`)
+    core.setOutput('url', resp.data.url)
+    core.setOutput('url_html', resp.data.html_url)
```
Comment (dorny) [inline src/main.ts:198]: @peterwhitesell I've just noticed this is executed in a loop. Multiple check runs might be created if the input path has a pattern matching more files. In that case, the `url` variable would be set to the last one. What's your use case? We could leave it as is or output JSON encoded array instead. Not sure what's a better approach.

File: src/main.ts [hunk 2]
```diff
@@ -211,6 +214,8 @@ class TestReporter {
         return new JestJunitParser(options)
       case 'mocha-json':
         return new MochaJsonParser(options)
+      case 'go-junit':
+        return new GoJunitParser(options)
```
Comment (dorny) [inline src/main.ts:218]: We should have go-junit support documented in the README and action.yml

**Review by dorny — CHANGES_REQUESTED** [review]

**Comment by sonomirco** [comment]
Hi @dorny and @peterwhitesell
This is precisely what I am looking for! 
I noticed the HTML  <github repo project>/runs/<random number>
Could this directory be fixed? So I can link it to a GitHub Env in a GitHub action doing something like this.

    environment:
      name: unit-test
      url: <report-url>

**Comment by sonomirco** [comment]
@peterwhitesell and @dorny seems this PR is not moving forward.
Do you have any updates?

**Comment by luisito666** [comment]
Hi i am the original developer of this PR, i plan to resend this today

**Comment by luisito666** [comment]
I am the original developer of this feature and i decide to create another PR
Here is the PR https://github.com/dorny/test-reporter/pull/242

**Comment by j-catania** [comment]
#242 Merged.
Thanks both

**Pull request closed without merging**